### ISSUE TITLE
Add a Proper Vertical Split

### ISSUE THREAD

**Issue by Archie-2021** [gitobsidiantutorial/obsidian-tabs#20]
Like how it is now for "proper horizontal split" please add one for a vertical one as well because it is even more practical. Also it would be nice if tab bar don't go away in double note/split mode. one can put the focus on one pane and select a note from the tab bar to get it. I think it is very useful as I usually use two panes.

And thanks for your effort for this great plugin

**Comment by thatdwayne** [comment]
Agreed - I tried following the instructions, but cannot get a vertically split, 2-pane view. I love this plugin otherwise - thanks for making it!

**Comment by jroachgit** [comment]
I'm not having any luck with multiple windows at all via vertical / horizontal splitting.

**Comment by ironhak** [comment]
Up.

**Comment by HuangAdam** [comment]
+1

**Comment by ClandestineClout** [comment]
So there's currently no way to split panes vertically to have them side by side with this plugin enabled? I click split vertically and it does nothing. I click split horizontally and it shows one 0ane on the left and two panes stacked horizontally on the right, obscuring half the left pane, creating what looks like a real mess in obsidian.
